composer.json

```diff
@@ -25,7 +25,7 @@
     },
     "config": {
         "platform": {
-            "php": "7.2.5"
+            "php": "7.2.9"
```

Comment:
is this upgrade necessary? Does it mean we no longer support 7.2.5 as a minimum version @sgiehl ?

Reply:
That's the platform override value. So composer assumes 7.2.9 is installed, but the requirement is still set to 7.2.5

Reply:
👍

Reply:
@tsteur we maybe should merge this if possible before releasing a new version, as there might otherwise be issues on PHP 8

Reply:
@sgiehl planning to merge in a bit. https://github.com/wikimedia/less.php/blob/master/composer.json#L21 mentions it requires 7.2.9 so hoping it will still run on 7.2.5 (was changed in https://github.com/wikimedia/less.php/commit/28baca4fb84da050f8d7f625230753d5ca461bca)

Reply:
see https://github.com/wikimedia/less.php/issues/50#issuecomment-747566164